lib/PlatoonService/src/PlatoonController.h

```diff
@@ -133,6 +135,16 @@ class PlatoonController
      */
     static const int32_t FORWARD_CONE_APERTURE = 1300; /* Aprox. 1/3 * pi */
 
+    /**
+     * Distance interval between waypoints in mm.
+     */
+    static const int32_t WAYPOINT_DISTANCE_INTERVAL = 50;
```

Comment:
Can it be negative or is the calculation later with signed data types?

Reply:
Calculation is later with signed data types (PlatoonController.cpp:159)